services/gam/src/lib.rs

```diff
@@ -43,6 +43,7 @@ pub const APP_NAME_SHELLCHAT: &'static str = "shellchat";
 pub const APP_MENU_NAME: &'static str = "app menu";
 pub const KBD_MENU_NAME: &'static str = "keyboard menu";
 pub const WIFI_MENU_NAME: &'static str = "WLAN menu";
+pub const APP_NAME_MTXCLI: &'static str = "mtxcli";
```

Comment:
since you're submitting a PR for the main repo, you don't need this line. It will automatically be generated if you do `cargo xtask app-image mtxcli` or `cargo xtask run mtxcli`.

Reply:
I meant to make this a review comment.

Reply:
Removed